app/components/webauthn_input_component.rb

```diff
@@ -22,10 +22,16 @@ def call
       :'lg-webauthn-input',
       content,
       **tag_options,
-      hidden: true,
-      platform: platform?.presence,
-      'passkey-supported-only': passkey_supported_only?.presence,
+      **initial_hidden_tag_options,
       'show-unsupported-passkey': show_unsupported_passkey?.presence,
     )
   end
+
+  def initial_hidden_tag_options
+    if platform? && passkey_supported_only?
+      { hidden: true }
+    else
+      { class: 'js' }
```

Comment:
I probably should have clarified how this is working: We have [`.js` and `.no-js` class styles](https://github.com/18F/identity-idp/blob/ba0bdd0d82501b184e467f3edfc4a19c1d252c51/app/assets/stylesheets/utilities/_util.scss#L3-L9) which hide the content in JavaScript disabled / enabled environments respectively ([applied](https://github.com/18F/identity-idp/blob/ba0bdd0d82501b184e467f3edfc4a19c1d252c51/app/views/layouts/base.html.erb#L2) and [toggled](https://github.com/18F/identity-idp/blob/ba0bdd0d82501b184e467f3edfc4a19c1d252c51/app/views/layouts/base.html.erb#L23-L25) in the base layout). So by adding `js` as a class here,  it's how we hide the content for browsers with JavaScript disabled.